### ISSUE TITLE
Do we still need URN subclassifications.

### ISSUE THREAD

**Issue by peterbrightwell** [AMWA-TV/nmos-discovery-registration#161]
This came up in review for proposed v1.3.1

`2.1. APIs - Common Keys.md` mentions possibly using future format subclassifications such as `urn:x-nmos:format:video.raw`.  Should this be removed/deprecated, as in practice we use IANA `media_type`?

**Comment by peterbrightwell** [comment]
Are there any examples of other subclassifications that are actually used that would be more suitable here?  Is the .ucast / .mcast example used in practice?

**Comment by garethsb** [comment]
Yes it is (though causes pain, e.g. Query API Basic Query for `transport=urn:x-nmos:transport:rtp` _doesn't_ return the subclassifications because it uses exact string match)

**Comment by peterbrightwell** [comment]
We made further improvements for `v1.3.1` release. See #166
